#!/usr/bin/env python
PACKAGE = "control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("p_steering",     double_t, 0, "p",      0, 0, 10)
gen.add("i_steering",     double_t, 0, "i",      0, 0, 10)
gen.add("d_steering",     double_t, 0, "d",      0, 0, 10)
gen.add("i_min",          double_t, 0, "i min",  0, -10, 10)
gen.add("i_max",          double_t, 0, "i max",  0, -10, 10)
gen.add("use_antiwindup",       bool_t, 0, "use_antiwindup", True)

exit(gen.generate(PACKAGE, "steering_controller", "SteeringController"))
